test/e2e/util/util.go

```diff
@@ -1312,7 +1312,11 @@ func ValidatePrivateCluster(t *testing.T, ctx context.Context, client crclient.C
 		g.Expect(hostedCluster.Status.ControlPlaneEndpoint.Host).ToNot(ContainSubstring("hypershift.local"))
 	}
 
-	validateHostedClusterConditions(t, ctx, client, hostedCluster, numNodes > 0, 10*time.Minute)
+	// On OpenStack, the console operator is not Ready because Ingress is not available yet.
+	// This makes this validation fail, so we skip it for now.
+	if hostedCluster.Spec.Platform.Type != hyperv1.OpenStackPlatform {
+		validateHostedClusterConditions(t, ctx, client, hostedCluster, numNodes > 0, 10*time.Minute)
```

Comment:
Note for myself: because we were waiting for 10 minutes here, I wonder if the validations after this one might fail because they're run too soon.

Reply:
`validateHostedClusterConditions` builds the set of conditions it needs to validate based on the `hostedCluster` - please amend the logic there, not here.

Reply:
eg https://github.com/openshift/hypershift/blob/c65722a26b5ace2b92eab2e41f5e2b18163b660f/support/conditions/conditions.go#L35-L72

Reply:
thanks, I'll take a look here.